site/content/en/docs/commands/pause.md

```diff
@@ -7,11 +7,12 @@ description: >
 
 ## minikube pause
 
-pause Kubernetes
+Pauses a running local Kubernetes cluster
```

Comment:
I think it is already implied that this is a local Kubernetes,
How about, 
“Pauses the api-server for the active Kubernetes cluster”